docs/js/features/odata/odata-v2-client.md

```diff
@@ -44,7 +46,7 @@ Each request type has additional request specific configuration options:
 
 Creating  an entity `asChildOf` another entity for [`create`](#create-request-builder) requests, [ETag handling](#handling-of-etags) for [`update`](#update-request-builder) and [`delete`](#delete-request-builder) requests, as well as set operations for [`getAll`](#getall-request-builder) requests and [`select`](#select)ing properties for [`getAll`](#getall-request-builder) and [`getByKey`](#getbykey-request-builder) requests.
 
-The last step when making a request using the SAP Cloud SDK is the request execution. Once the request is configured, chain the `execute` function and pass a [destination](/cloud-sdk/docs/js/features/connectivity/destination-js-sdk/) to it.
+The last step when making a request using the SAP Cloud SDK is the request execution. Once the request is configured, chain the `execute` function and pass a <a href={useBaseUrl('docs/js/features/connectivity/destination-js-sdk')}>destination</a> to it.
```

Comment:
The difference is, that we now use `useBaseUrl` which automatically adds the baseUrl in front (which could be different in different environments). That is what allows us to change the baseUrl in the config to '/' and run this locally. (And doing this is necessary because of a bug in docusaurus serve, but anyways this is cleaner).